airflow/providers/databricks/hooks/databricks.py

```diff
@@ -343,6 +343,17 @@ def get_run_output(self, run_id: int) -> dict:
         run_output = self._do_api_call(OUTPUT_RUNS_JOB_ENDPOINT, json)
         return run_output
 
+    async def a_get_run_output(self, run_id: int) -> dict:
+        """
+        Retrieves run output of the run.
```

Comment:
I would add a comment about single task run. Because it's a common problem when people are trying to fetch a top-level run, but it should be a single task inside the job: https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunsCancelAll

Reply:
removed it and restore the `a_get_run_state` call to get a state in the trigger for the scope of this PR.